### PR TITLE
Feature/fiat withdrawal form

### PR DESCRIPTION
**Pull request by letehaha**

### CODE REVIEW

File: src/vue/modules/withdrawal-fiat-bank/index.vue [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    fees: {
+      fixedFee: EMPTY_FEE,
+      percentFee: EMPTY_FEE,
+    },
+    form: {
+      asset: null,
+      amount: null,
+      userName: null,
+      bankName: null,
+      iban: null,
```
Comment (max-nes) [inline src/vue/modules/withdrawal-fiat-bank/index.vue]: I believe BIC/SWIFT code field is missing?

File: src/vue/modules/withdrawal-fiat-bank/index.vue [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+        },
+      },
+    }
+  },
+  async created () {
+    initApi(this.wallet, this.config)
+
+    this.setAccountId(this.wallet.accountId)
+    await this.loadBalances()
+    this.isInitialized = true
```
Comment (max-nes) [inline src/vue/modules/withdrawal-fiat-bank/index.vue]: Better to move it to the very bottom of the created()

File: src/vue/modules/withdrawal-fiat-card/index.vue [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+        },
+      },
+    }
+  },
+  async created () {
+    initApi(this.wallet, this.config)
+
+    this.setAccountId(this.wallet.accountId)
+    await this.loadBalances()
+    this.isInitialized = true
```
Comment (max-nes) [inline src/vue/modules/withdrawal-fiat-card/index.vue]: the same stuff, let’s move it to the bottom

File: src/vue/modules/withdrawal-fiat-bank/index.vue (hunk 1 above)
Comment (letehaha) [reply src/vue/modules/withdrawal-fiat-bank/index.vue]: Used all fields that Aleksandr Plotnikov send. Do we need to add a `BIC/SWIFT ` too?

Comment (max-nes) [reply src/vue/modules/withdrawal-fiat-bank/index.vue]: @letehaha yep

**Pull request merged**